data/shortcut-overlay.appdata.xml.in

```diff
@@ -10,7 +10,7 @@
     <p></p>
   </description>
   <releases>
-    <release version="2.0.0" date="2022-05-08" urgency="low">
+    <release version="2.0.0" date="2022-05-09" urgency="low">
```

Comment:
How strict should we be with date accuracy? At this point it should be the 10th if we want complete accuracy